OpenXmlPowerTools/SmlDataRetriever.cs

```diff
@@ -936,7 +1080,7 @@ private static void MoveBooleanAttribute(XElement props, XElement xf, XName attr
             "System Background",
         };
 
-        private static string[] FontFamilyList = new string[] {
+        private static readonly string[] FontFamilyList = new string[] {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the array type; it is redundant.](https://app.codacy.com/gh/Codeuctivity/Open-Xml-PowerTools/pullRequest?prid=5308324)